_includes/menu.html

```diff
@@ -4,79 +4,49 @@
-            {% assign basedir = page.url | remove_first: '/' | split: '/' | first | lstrip %}
-            {% assign linkdir = nav_item.url | replace: "/", "" | lstrip %}
-          {% endif %}
         <li class="usa-nav__primary-item">
+          {% assign nav_item_class = "" %}
+          {% assign current_item  = "" %}
+          {% if nav_item.url == page.url %} 
+          {% assign current_item = "usa-current" %}
+        {% endif %}
+          {%- if nav_item.title contains 'Contact' -%}
+            {% assign nav_item_class = "usa-button an18f-button--dark margin-top-3 desktop:margin-top-0 desktop:margin-x-2"  %}
+          {%- else -%} 
+            {% assign nav_item_class = "usa-nav__link" %}
+          {%- endif -%} 
           <a
-            class="usa-nav__link{% if basedir == linkdir %} usa-current{% endif %}"
+            class="{{ nav_item_class }} {{current_item}}"
             href="{{ nav_item.url | url }}"
-            ><span>{{ nav_item.name | escape }}</span></a
-          >
-        </li>
-        {% else %} {% assign nav_id = 'primary-nav-' | append: forloop.index %}
-        <li class="usa-nav__primary-item">
-          <button
-            class="usa-accordion__button usa-nav__link"
-            aria-expanded="false"
-            aria-controls="{{ nav_id }}"
-            type="button"
           >
-            <span>{{ nav_item.name | escape }}</span>
-          </button>
-          <ul id="{{ nav_id }}" class="usa-nav__submenu">
-            {% for subnav_item in nav_item.children %}
-            <li class="usa-nav__submenu-item">
-              <a href="{{ subnav_item.url | url }}"
-                >{{ subnav_item.name | escape }}</a
-              >
-            </li>
-            {% endfor %}
-          </ul>
+            <span>{{ nav_item.title | escape }}</span>
+          </a>
         </li>
-        {% endunless %} {% endfor %}
+        {% endfor %}
       </ul>
-      {% comment %}
-      TODO: The conditional and bottom-2 class on the line below can be removed after launch.
-      It's there to fix wonky alignment when search is present on the index page
-      {% endcomment%}
-      <div class="usa-nav__secondary {% unless tags %} bottom-2 {% endunless %}">
-        {% if navigation.secondary_navigation %}
-        <ul class="usa-nav__secondary-links">
-          {% for nav_item in navigation.secondary_navigation %}
-          <li class="usa-nav__secondary-item">
-            <a href="{{ nav_item.url | url }}" class="text-bold">
-              {{ nav_item.name | escape }}
-            </a
-            >
-          </li>
-          {% endfor %}
-        </ul>
-        {% endif %}
-        {% include "searchgov/form.html" searchgov: site.searchgov %}
-      </div>
-    </div>
-  </nav>
+      <a href="https://search.usa.gov/search?utf8=%E2%9C%93&affiliate=18F-site&query=&commit="
+          class="usa-button padding-x-2"
+      >
+        <img src="/assets/uswds/img/usa-icons-bg/search--white.svg"
```

Comment:
The link checker failed to find the asset as written.

Use the `image` shortcode for this asset, I think it would be:
```
{% image "assets/uswds/img/usa-icons-bg/search--white.svg" "Search" %}
```
(No leading slash)

Also for future reference, `npm run precommit` will run all the build checks locally.

Reply:
Fixed. Thanks for the tip!